apps/core/views.py

```diff
@@ -90,6 +92,10 @@ def get(self, request, *args, **kwargs):
                              pixels_by_omics_area],
                 },
 
+                'number_of_pixels': {
+                    'title': _('Number of Pixels'),
+                    'data': number_of_pixels,
+                },
```

Comment:
well, actually, you should directly return the `number_of_pixels` here, no need for a dict since it is just a number and you dont draw a chart.